families/burrow_evm/src/sawtooth_burrow_evm/cli/exec.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"github.com/jessevdk/go-flags"
+	client "sawtooth_burrow_evm/client"
+	sdk "sawtooth_sdk/client"
+)
+
+type Exec struct {
+	Private string `short:"k" long:"key" required:"true" description:"A hex encoded private key to make the message call with."`
+	To      string `short:"t" long:"to" required:"true" description:"A 160 bit hex encoded address of an account to call."`
+	Data    string `short:"d" long:"data" description:"Hex encoded data to call the account with."`
+	Url     string `short:"U" long:"url" description:"The REST API URL to connect to when sending the transaction." default:"http://127.0.0.1:8080"`
+	Gas     uint64 `short:"g" long:"gas" description:"Amount of gas to supply the transaction with." default:"1000"`
+}
+
+func (e *Exec) Name() string {
+	return "exec"
+}
+
+func (e *Exec) Register(p *flags.Parser) error {
+	_, err := p.AddCommand("exec", "Call an existing account", "", e)
+	return err
+}
+
+func (e *Exec) Do() error {
```

Comment:
`Do` is not a great name for this.   `Execute`,  or `Run` are far better names here.